src/App.jsx

```diff
@@ -159,7 +159,7 @@ const App = (props) => {
 
                 <Route
                   path="/:language/contract-20/"
-                  element={<Contract is30ContractEnabled={false} {...props} />}
+                  element={<Contract {...{...props,is30ContractEnabled:false}} />}
```

Comment:
Why? It is way harder to understand. Maybe because props overrided is30ContractEnabled? What about just changing the order?